router/index.js

```diff
@@ -2,7 +2,11 @@ import express from 'express';
 const router = express.Router();
 
 import userRouter from './userRouter';
+import cartRouter from './cartRouter';
+import kakaoRouter from './kakaoRouter';
 
 router.use('/users', userRouter);
+router.use('/cart', cartRouter);
+router.use('', kakaoRouter);
```

Comment:
```suggestion
router.use('/users', userRouter);
router.use('/carts', cartRouter);
router.use('', kakaoRouter);
```

url 단/복수 통일 해주세요 :) 복수형이 좋게네욤

Reply:
네 소헌님 이 부분 수정해서 다시 push 완료했습니다 !